src/api/block.rs

```diff
@@ -10,9 +10,10 @@ use crate::{
 use concordium_rust_sdk::{
     common::SerdeSerialize,
     types::{BakerId, SpecialTransactionOutcome},
-    v2::IntoBlockIdentifier,
+    v2::{IntoBlockIdentifier, Upward},
```

Comment:
please update the changelog with a description how `unkowns` are handled in this project.